pkg/catalog/helm/helm.go

```diff
@@ -23,21 +25,30 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var httpTimeout = time.Second * 10
```

Comment:
Can we change this to 30 seconds? I think this value might be a common practice.

Reply:
yes